### ISSUE TITLE
Error config version 1.7.7476

### ISSUE THREAD

**Issue by theusmpa** [jaskie/PlayoutAutomation#83]
Hello!

I went to test the latest version (1.7.7476), however I was unable to make the settings, I have the following error below when clicking on "Play-out-servers" and "Rundown-engines" in the configuration application:

![erroTvPlay](https://user-images.githubusercontent.com/6371447/85575315-d6a21d00-b60d-11ea-8b79-6fb248c53ccb.png)

What is the solution?

In the previous version (1.7.7369), it is working.

tks

--Matheus

**Comment by jaskie** [comment]
Hi,
I can't reproduce this. It's on MySql or SQLite database? If it's on MySql, database have to be created inside the configuration window. If it's on SQLite, try removing the database. It exists in `%localappdata%\TVPlay\TVPlay.db` file, so rename it - the config program should create it new, empty. You can send me the invalid one for testing.

**Comment by theusmpa** [comment]
I'm trying to install on a new machine (will be the backup), machine zero, newly formatted windows, already installed all the necessary dependencies according to the wiki.

When checking the database configuration, it does not show me any options:

![erro2](https://user-images.githubusercontent.com/6371447/85610696-9bafe180-b62d-11ea-9f22-b8ae8d4d86b1.png)


Any idea what might be going on?

The same thing happened in the previous version :(

tks

--Matheus

**Comment by jaskie** [comment]
Are TAS.Database.MySqlRedundant.dll and TAS.Database.Sqlite.dll in the same directory, as TVPlay.config.exe? If so, try to remove the latter from the dir.
It may be that SQLite have some dependencies I do not identified, and it prevents SQLite loading. It for sure depends on System.Data.SQLite.dll and x64\SQLite.Interop.dll. But the last one? I do not know now...

**Comment by theusmpa** [comment]
Strange that if I download the latest version on the same machine that is running the previous version, and try to run, the same problem happens ... I have no database option ...
I go back to the version that works and the database options appear again.
I don't know what could be wrong!
Any tests I can do?

tks

--Matheus

**Comment by jaskie** [comment]
Did you tried TAS.Database.Sqlite.dll removal?

**Comment by jaskie** [comment]
No, it's neither MySQL nor SQLite, probably the .NET runtime itself loads the C++ runtime. However, try to remove TAS.Database.SQLite.dll and Tas.Database.MySqlRedundant.dll one or another - check if remained provider is available.
And ensure that no exe or dll files remains from previous version, it would be the reason.
Also try to remove language folders, the problem may stem there (w-o lanaguage files all UI will be in English).

**Comment by theusmpa** [comment]
Nothing worked! I can run only one version that I've been using for a while (v1.7.7369), if I download the same version from github and try to run it presents the same behavior as the last version (v1.7.7476), all the tests done on two different machines. .

To make it more difficult, I do not remember the date I downloaded the only version I can run, but it is version 1.7.7369 which presents that bug of not keeping the user's administrator permission settings saved.

I don't know what else to do, any other ideas?

**Comment by jaskie** [comment]
So it's my turn. I'll prepare a test environment and try to reproduce the issue.

**Comment by theusmpa** [comment]
OK thank you!

I am waiting for the result, in the meantime I will test formatting the machine and installing another version of windows, maybe try a version of windows server, because currently I use Windows 10 Pro ...

Do you think there can be something?

I also have some "device buffer" messages on the decklink in the CasparCG logs, I think I saw somewhere that the windows server does not give these problems ....

I will give back as soon as possible!

**Comment by jaskie** [comment]
Would you provide me exact Windows version you are installing TVPlay on?
I just tested on newly installed Windows 10 Pro x64 2004, with English and (additionally installed) Spanish (Latin America), and everything works smoothly. Tested also after vc++ runtime was installed.
Now I'm installing x86 version...
Regarding "device buffer" messages - if they are with [trace] or [information] severity level, just ignore them or change log level in casparcg.config.

**Comment by theusmpa** [comment]
The errors present at the trace level, so I will ignore them.
Sometimes I also have a lot of frame errors late in Deckling (warning level), any tips?

About the windows version I'm running Windows 10 Pro x64 10.0.18363 compilation 18363, lingua pt-br (portugues brasil).

**Comment by theusmpa** [comment]
What is the exact version of the c ++ runtime you should install? Maybe my error could be there, I'm looking at it has several versions installed on the machine, I don't know if that would cause problems ...

**Comment by jaskie** [comment]
The config app should work (and works in my case) without anything other installed. I'm downloading 19H2 version of Windows, as every clean installation I today tested was 2004. Strange, however. It can be only the Windows or language version dependent.

**Comment by jaskie** [comment]
@theusmpa, I just verified that using the [latest release](https://github.com/jaskie/PlayoutAutomation/releases/tag/1.7.7476), under fresh, clean Windows installation:

- x64 1909 en-us,
- x64 2004 en-us,
- x64 2004 pt-br,
- x86 2004 en-us

the problem does not occur, so I'm closing the issue. I can provide you further assistance using some remote desktop means of your choice and wish.

**Comment by theusmpa** [comment]
@jaskie ,  I thank you so much for your efforts, it has already helped a lot, now it's up to me, I will try on a version of windows like this that you were successful .

Thank you very much for your time and dedication invested in this problem.

**Comment by jaskie** [comment]
Don't hesitate to reopen this issue if you find a clue. I may be wrong, nobody's perfect...

**Comment by theusmpa** [comment]
Find out what was going on!

I was copying the files already unzipped from my work notebook, but I noticed that in the properties of all files they had a security option to unlock the file ... that is to say ... for some reason still unknown, everything did that unzipped the file and I take it in a folder for the servers, all files are blocked, even allowing to run the .dll files for the first time and the other dependencies are blocked ....

See the image below:

Solution: I copied the zipped file and unzipped it on the server ...... done, problem solved!

However I found a new problem: after performing all the proper configuration, when trying to run tvPlay, nothing happens! It doesn't open. In the windows task manager I can see that it appears and disappears. It doesn't even leave a log.
The same happens with TVplayClient

I installed, reinstalled, changed the version of the runtine c ++ and nothing resolved!

Any tips? Did you not observe this behavior in your tests on a clean Windows installation?

I observed this behavior on a clean installation of Windows 2004 en-us.

1.7.7369 is functioning normally. This happens only in version 1.7.7.7476.

Tks

--Matheus
